backend/src/routes/api/storage/index.ts

```diff
@@ -0,0 +1,65 @@
+import { FastifyInstance, FastifyReply } from 'fastify';
+import { createMinioClient, getObjectStream } from './storageUtils';
+import { getDashboardConfig } from '../../../utils/resourceUtils';
+import { getDirectCallOptions } from '../../../utils/directCallUtils';
+import { getAccessToken } from '../../../utils/directCallUtils';
+import { OauthFastifyRequest } from '../../../types';
+
+export default async (fastify: FastifyInstance): Promise<void> => {
+  fastify.get(
+    '/:namespace/:bucket',
```

Comment:
Remove bucket, get from DSPA -- maybe make `createMinioClient` into a `setupMinioClient` function that returns the bucket and the client... then use them in the `getObjectStream`.